### ISSUE TITLE
Mobile support

### ISSUE THREAD

**Issue by valentinedwv** [Esri/geoportal-server-catalog#228]
Support for tablets, and possibly phones.

----
Note from a user.
I tried datadiscoverystudio.org on an iPad and got the attached screen.

**Comment by mhogeweg** [comment]
could you expand on this use case? what would you do with the thing you found in a data portal when on a mobile device?

**Comment by valentinedwv** [comment]
We will work on one.

Many people run around with tablets, or phablets. So demonstrating the product at a conference without having to whip out a computer.

**Comment by valentinedwv** [comment]
Potrait on tablets may be the issue. Attaching a dev shot of rotated tablet from apple safari.

Is there a way to say, rotate your screen
![image](https://user-images.githubusercontent.com/851784/50020073-b493fd00-ff89-11e8-95a3-f43bb74d15ab.png)

**Comment by KathleenPate** [comment]
I designed the portal to work on a tablet using the bootstrap CSS. We're not in production so I can't show you. 
![portal-screencapt](https://user-images.githubusercontent.com/13800356/50020203-7e0eb000-ff92-11e8-8f6e-0d4c88180ada.png)

**Comment by valentinedwv** [comment]
Would love to see those changes.

**Comment by KathleenPate** [comment]
The Sandbox site displays with the iPad simulator. I found, like you, that the menu expanded and the search results slipped underneath the left menu after I had made some modifications to the original code. I put the logo into its own row to simplify layout clashes with the menu.  I used browser Inspectors to made sure the site had a `@media` designation. I made a few adjustments: I added to the navar-nav,  and modified the measurements for the .g-search-results class from pixels to **_relative_ measurement**. Here is what I did.
 Double check _calcite-bootstrap.css:_ line 3823 for the at media modifications. 
http://geoss.esri.com/geoportal2/lib/calcite-bootstrap/css/calcite-bootstrap.css
       `@media (min-width: 768px) .navbar-nav {float:left}`
I _added_ a phone modification in custom.css for the menu bar (phones rotated landscape work)
```
	@media (min-width: 768px) .navbar-nav {  margin: 5px; width: 70%; float: left;}
        .navbar-nav > li > a  {   display: inline; }
```

**in main.css :** 
```
.g-search-results {
  max-height: -webkit-calc(100vh - 16em);
  max-height:    -moz-calc(100vh - 16em); 
  max-height:         calc(100vh - 16em);
  overflow: auto;
}
```
![portal-screencapt-phone](https://user-images.githubusercontent.com/13800356/50104825-3b461580-01e8-11e9-95d4-ca111e8598aa.png)
